pkg/storaged/filesystem/mounting-dialog.jsx

```diff
@@ -42,9 +42,9 @@ import {
 
 const _ = cockpit.gettext;
 
-export function mounting_dialog(client, block, mode, forced_options) {
+export function mounting_dialog(client, block, mode, forced_options, subvol) {
```

Comment:
How would I reproduce this? Do I already need it mounted?

Reply:
So I suppose have it mounted and:

![image](https://github.com/cockpit-project/cockpit/assets/67428/1a085198-4ab5-4e40-8857-c17f30dd108b)

Reply:
This also shows the custom mount options hm which I kinda like but I guess you rather hide as it's an implementation detail?

Reply:
Right reproduced this with a mess fstab

```
mount: /mnt/cake: /dev/sda already mounted on /run/butter.
```

```
/dev/sda on /run/butter type btrfs (rw,relatime,space_cache=v2,subvolid=5,subvol=/)
/dev/sda on /run/cake type btrfs (rw,relatime,space_cache=v2,subvolid=5,subvol=/)
/dev/sda on /mnt/foo type btrfs (rw,relatime,space_cache=v2,subvolid=5,subvol=/)
/dev/sda on /mnt/cake type btrfs (rw,relatime,space_cache=v2,subvolid=5,subvol=/)
```

So we take the device ID, it would be nicer to saw btrfs volume X but honestly it's not a super big deal to me.

Reply:
The duplicate mount point stuff should be fixed now.

Reply:
(I want to resolve this...)

Reply:
I think this is resolved, but Github doesn't let me formally resolve this thread...